internal/cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -221,15 +218,14 @@ func (c bundleCmd) runManifests() (err error) {
 		return fmt.Errorf("error writing bundle scorecard config: %v", err)
 	}
 
-	if !c.quiet && !c.stdout {
-		fmt.Println("Bundle manifests generated successfully in", c.outputDir)
-	}
+	c.println("Bundle manifests generated successfully in", c.outputDir)
 
 	return nil
 }
 
 // writeScorecardConfig writes cfg to dir at the hard-coded config path 'config.yaml'.
 func writeScorecardConfig(dir string, cfg v1alpha3.Configuration) error {
+	// Skip writing if config is empty.
```

Comment:
i know we're trying to get this in for 1.5 but this is the sort of thing a test would be really nice for, instead of a comment here in the source.

Reply:
While I agree, the behavior of this function doesn't change in this PR.